net/openthread-br/Makefile

```diff
@@ -0,0 +1,60 @@
+#
+#  Copyright (c) 2021, The OpenThread Authors.
+#  All rights reserved.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=openthread-br
+PKG_RELEASE:=1
+PKG_VERSION:=0.3.0
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/openthread/ot-br-posix.git
+PKG_SOURCE_DATE:=2021-04-23
+PKG_SOURCE_VERSION:=5a3c3221d6ae6b087d0933d98657ba818ae6ff31
+PKG_MIRROR_HASH:=1ba73550142d9f41f1528b064e4ef85c9114fc2b0da3fa7fee528e300975bf8a
+
+PKG_MAINTAINER:=OpenThread Maintainers <openthread-maintainers@googlegroups.com>
```

Comment:
Who is responsible for it? Is there any GitHub account for that? I mean, if there are issues who can we tag to mention him? I don't think that you expect that we will be writing an email each time, right? :-)